### ISSUE TITLE
Transitioning of Adult PLHIV on First line ARV

### ISSUE THREAD

**Issue by chris-githu** [chap-uzima/DataWarehouse#9]
@lmwaniki 
Following this request, please confirm whether the following data elements `for active ART clients` will suffice

Data Element | Notes
-- | --
Facility Name |
PatientID |
DOB |
Gender |
AgeLastVisit | >=15
CurrentRegimen | _TDF-3TC-NVP or AZT-3TC-EFV or TDF-3TC-EFv400mg_
LastVL | 
LastVLDate | 

@vogoti Do you need a query to run at sites?
FYI @irenefaith

**Comment by lmwaniki** [comment]
@chris-githu  I would also take this as an opportunity to clean our EMR on regimens thus my query would not restrict age...

**Comment by vogoti** [comment]
@chris-githu ,Yes that would come in handy,as @lmwaniki mentioned the naming of the regimens is not standard for our facilities,is there a way you can include all different namings for the regimens so that it can also be run centrally

**Comment by vogoti** [comment]
@chris-githu as we continue preparing for upload June DBs could you please provide us with the query to pull this data so that we can have it run at facility level for our 5 Nairobi facilities,we can run for the other facilities once we upload all our June DBs
FYI @lmwaniki  @irenefaith @dgaitho
